spec/services/catalog/copy_portfolio_item_spec.rb

```diff
@@ -30,7 +30,7 @@
 
         expect(new.description).to eq portfolio_item.description
         expect(new.owner).to eq portfolio_item.owner
-        expect(new.name).to match(/^Copy of.*/)
+        expect(new.name).to eq portfolio_item.name
```

Comment:
Do we have a spec in here where `^Copy of .*` is tested when copied into a different `portfolio_item`?

Reply:
Yep, right here https://github.com/ManageIQ/catalog-api/blob/d9008c798b2208120dfd15bccc569c4a232796f1/spec/services/catalog/copy_portfolio_item_spec.rb#L20-L22

This test just makes sure the name wasn't modified. I'll put that specific case into a different it-statement so it's more apparent.